package.json

```diff
@@ -98,7 +98,7 @@
     "check:tls": "mocha --opts '{}' test/manual/tls_support.test.js",
     "format": "npm run check:lint -- --fix",
     "release": "standard-version -i HISTORY.md",
-    "test": "npm run lint && mocha --recursive test/functional test/unit"
+    "test": "npm run check:lint && mocha --recursive test/functional test/unit"
```

Comment:
Has this just been broken? OMG

Reply:
Yeah not for too long though, I think since the unified test runner was merged.